autodepgraph/graph.py

```diff
@@ -56,6 +56,10 @@ def __init__(self, name, cfg_plot_mode='svg',
         self._exec_cnt = 0
         self._calib_cnt = 0
         self._check_cnt = 0
+        
```

Comment:
These look like hardcoded defaults,  I think a comment should be added what these are (or why these values are chosen). One could argue that this should be an (optional) keyword argument so that the user can change this but I think simpler is probably better in this case.

Reply:
@AdriaanRol The `matplotlib_edge_properties` can be updated by the user to set properties of the plotting. These values are chosen such that when the user does not explicitly set them, the graphs will look like before

Reply:
Could you add that comment to the code in some form?

Reply:
@AdriaanRol Done!